src/ashtha/Assignment15/BankAccount.java

```diff
@@ -0,0 +1,209 @@
+/*Assignment 15 - 9th April
+ * Bank program by taking input from the user */
+package ashtha.Assignment15;
+
+import java.util.Scanner;
+
+public class BankAccount {
+	double iciciBalance, hdfcBalance, bobBalance, sbiBalance;
+	int iciciDebitCount = 0, hdfcDebitCount = 0, bobDebitCount = 0 , sbiDebitCount = 0;
+	int iciciCreditCount = 0, hdfcCreditCount =0, bobCreditCount =0, sbiCreditCount =0 ;
+	static int totalDebitCount = 0;
+	static int totalCreditCount = 0;
+	double totalBalance;
+	
+//Method to perform bank operation (credit or debit) as per user selection
+	
+	void transaction (String bank,String operation, double amount) {
+		bank = bank.toUpperCase();
+		operation = operation.toUpperCase();
+		switch(operation) {
+			case "DEBIT":
+				switch (bank) {
+					case "ICICI":
+						if (amount<= iciciBalance) {
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.